scripts/src/main/resources/scripts/functions

```diff
@@ -744,7 +753,7 @@ function doInstallWithPackageManager() {
 # $1: filename (e.g. ${filename})
 function doIsInstallerExtension() {
   local myext="${1/*\./.}"
-  if [ "${myext}" = ".msi" ] || [ "${myext}" = ".exe" ] || [ "${myext}" = ".pkg" ] || [ "${myext}" = ".bat" ] || [ "${myext}" = ".py" ]
+  if [ "${myext}" = ".exe" ] || [ "${myext}" = ".pkg" ] || [ "${myext}" = ".bat" ] || [ "${myext}" = ".py" ]
```

Comment:
See #978 this will break at least `az` commandlet, maybe also docker/rancher?
https://github.com/devonfw/ide-mirrors/blob/88a8f841b353913166c3809bfb593276841fdb67/az/ext-mappings
Can you add `noUnpack` to affected `commandlets` to keep their behavior unchanged.
Maybe this flag has to be set OS dependent...

Reply:
az do not use doInstall, it uses "doDevonCommand pip install azure-cli az".
No changes are necessary.
docker and rancher are not affected, because they use exe files for installation.